### ISSUE TITLE
put the container up on Dockerhub

### ISSUE THREAD

**Issue by bburns632** [uptake/groundhog#24]
> We'll put the container up on Dockerhub soon.

I would love to know what this is planned, and how I can help.

**Comment by jameslamb** [comment]
@bburns632 we need help from you, actually. If the Uptake Open Source Committee could provide a dockerhub account for `github.com/uptake` projects to use, we could do the work to publish to that account for this project, `autofocus`, `cran-server`, `airflow-api`, and `doc-server`.

I added the label `blocked` to be clear no one can pick this up themselves.

**Comment by bburns632** [comment]
Yes.  I've been brought up to speed on the issue.  The committee is working
on it.

**Comment by bburns632** [comment]
FYI This is unblocked.  We are on the beta for GitHub Package Registry, and the plan is to host and maintain packages there rather than DockerHub.
